package.nls.json

```diff
@@ -1,18 +1,18 @@
-    "settings.importStrategy.useBundled.description": "Always use the bundled version of `isort`.",
-    "settings.importStrategy.fromEnvironment.description": "Use `isort` from environment, fallback to bundled version only if `isort` is not available in the environment.",
-    "settings.interpreter.description": "When set to a path to Python executable, extension will use that to launch the server and any subprocess.",
-    "settings.showNotifications.description": "Controls when notifications are shown by this extension.",
-    "settings.showNotifications.off.description": "All notifications are turned off, any errors or warnings are still available in the logs.",
-    "settings.showNotifications.onError.description": "Notifications are shown only in the case of an error.",
-    "settings.showNotifications.onWarning.description": "Notifications are shown for errors and warnings.",
-    "settings.showNotifications.always.description": "Notifications are shown for anything that the server chooses to show."
+    "settings.args.description": "Arguments passed to isort for organizing imports in Python files. Each argument should be provided as a separate string in the array. <br> Example: <br> `\"isort.args\" = [\"--settings-file\", \"<file>\"]`",
+    "settings.check.description": "Whether to run `isort --check` on open Python files and report sorting issues as a \"Hint\" in the Problems window. You can update `isort.severity` to show sorting issues with higher severity. ",
+    "settings.severity.description": "Mapping of isort's message types to VS Code's diagnostic severity levels as displayed in the Problems window.",
+    "settings.serverEnabled.description" : "**Experimental** setting to control whether to run isort in a server-like mode. By default `isort` is run behind LSP server, but you can disable this setting to run isort directly. Disabling this setting will also disable import sorting via Code Actions or Organize Imports, but you can still sort imports through the **isort: Sort Imports** command.",
+    "settings.path.description": "Path of the isort binary to be used by this extension. Note: This may slow down formatting. <br> Examples: <br>- `isort.path : [\"~/global_env/isort\"]` <br> - `isort.path : [\"conda\", \"run\", \"-n\", \"lint_env\", \"python\", \"-m\", \"isort\"]`. ",
+    "settings.importStrategy.description": "Defines which isort binary to be used to lint Python files. When set to `useBundled`, the extension will use the isort binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the isort binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid isort binary in the selected environment, it will fallback to using the isort binary that is shipped with the extension The `pylint.path` setting may also be ignored when this setting is set to `fromEnvironment`. ",
```

Comment:
```suggestion
    "settings.importStrategy.description": "Defines which isort binary to be used to lint Python files. When set to `useBundled`, the extension will use the isort binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the isort binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid isort binary in the selected environment, it will fallback to using the isort binary that is shipped with the extension The `isort.path` setting may also be ignored when this setting is set to `fromEnvironment`. ",
```